react/styles/styles.css

```diff
@@ -0,0 +1,113 @@
+.videoContainer:fullscreen .videoElement {
+  width: 100%;
+  height: 100%;
+}
+
+.controlsContainer {
+  filter: contrast(0.9);
+}
+
+.videoElement {
+  object-fit: cover;
+}
+
+.trackBar {
+  width: 100%;
+  background-color: transparent;
+  -webkit-appearance: none;
+}
+
+.trackBar:focus,
+.button:focus {
+  outline: none;
+}
+
+.trackBar::-webkit-slider-runnable-track {
+  background: #e2e4e6;
+  border: 0;
+  border-radius: 18px;
+  width: 100%;
+  height: 7px;
+  cursor: pointer;
+}
+
+.trackBar::-webkit-slider-thumb {
+  margin-top: -5px;
+  width: 15px;
+  height: 15px;
+  background: #134cd8;
+  border: 0;
+  border-radius: 50px;
+  cursor: pointer;
+  -webkit-appearance: none;
+}
+
+.trackBar:focus::-webkit-slider-runnable-track {
+  background: #f0f1f2;
+}
+
+.trackBar::-moz-range-thumb {
+  width: 15px;
+  height: 15px;
+  background: #134cd8;
+  border: 0;
+  border-radius: 50px;
+  cursor: pointer;
+}
+
+.trackBar::-ms-track {
+  background: transparent;
+  border-color: transparent;
+  border-width: 10px 0;
+  color: transparent;
+  width: 100%;
+  height: 7px;
+  cursor: pointer;
+}
+
+.trackBar::-ms-fill-lower {
+  background: #d4d7da;
+  border: 0;
+  border-radius: 36px;
+}
+
+.trackBar::-ms-fill-upper {
+  background: #e2e4e6;
+  border: 0;
+  border-radius: 36px;
+}
+
+.trackBar::-ms-thumb {
+  width: 15px;
+  height: 15px;
+  background: #134cd8;
+  border: 0;
+  border-radius: 50px;
+  cursor: pointer;
+  margin-top: 0px;
+  /*Needed to keep the Edge thumb centred*/
+}
+
+.trackBar:focus::-ms-fill-lower {
+  background: #e2e4e6;
+}
+
+.trackBar:focus::-ms-fill-upper {
+  background: #f0f1f2;
+}
+
+.volumeSlider {
+  width: 5rem;
+  margin-top: 0;
+}
+
+.button {
+  border: none;
+  background: none;
+  color: white;
+  cursor: pointer;
+}
+
+.button:fullscreen {
+  position: fixed;
+}
```

Comment:
Most of these styles should be in the `store-theme` and not here (actually, might be all of them), here you should put only the needed styles to make the component visible and functional, anything else should go in the app theme